ArtOfIllusion/src/artofillusion/ObjectTextureDialog.java

```diff
@@ -145,15 +144,7 @@ public ObjectTextureDialog(LayoutWindow parent, ObjectInfo objects[], boolean in
     
     for (Texture texture : PluginRegistry.getPlugins(Texture.class))
     {
-      try
```

Comment:
I see only reflection-dependent exceptions can be thrown there. But there are no more such code.